test/k8s/manifests/1.21/external_endpoint.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: discovery.k8s.io/v1beta1
```

Comment:
AFAIK EndpointSlice becomes stable in 1.21, any reason we are still using v1beta1 here?

https://kubernetes.io/docs/concepts/services-networking/endpoint-slices/